api/v1/runtime/runtime_agent_api.proto

```diff
@@ -359,3 +360,32 @@ message KubeLinterCheck {
   uint64 passed = 2; // Represented as bitmasks of passed checks.
   uint64 failed = 3; // Represented as bitmasks of failed checks.
 }
+
+message Process {
+  uint32 pid = 1;
+  uint64 start_time = 2;
+  uint32 ppid = 3;
+  uint64 parent_start_time = 4;
+  repeated string args = 5;
+  string filepath = 6;
+  uint64 exit_time = 7;
+}
+
+enum ProcessAction {
+  PROCESS_ACTION_UNKNOWN = 0;
+  PROCESS_ACTION_EXEC = 1;
+  PROCESS_ACTION_FORK = 2;
+  PROCESS_ACTION_EXIT = 3;
+}
+
+message ProcessEvent {
+  uint64 timestamp = 1; // Stored as unix timestamp in nanoseconds.
+  string container = 2;
```

Comment:
Maybe name this `container_id` because in other parts we have `container_name` so it's not clear what is this field.